backend/geonature/core/command/main.py

```diff
@@ -158,3 +158,22 @@ def default_config():
     frontend_defaults = GnGeneralSchemaConf().load({}, partial=required_fields)
     defaults = ChainMap(backend_defaults, frontend_defaults)
     print(toml.dumps(defaults))
+
+
+@click.option("--key", type=str, required=False, help="Return only the config of this key")
```

Comment:
```suggestion
@click.argument("key", type=str, required=False, help="Return only the config of this key")
```
Plus simple à l’usage, un argument optionnel au lieu d’une option ?

Reply:
oui